src/main/java/seedu/address/logic/commands/ImportCommand.java

```diff
@@ -31,40 +33,44 @@
             + "Parameters: FILENAME\n"
             + "Example: " + COMMAND_WORD + " t35.json";
 
-    public static final String MESSAGE_SUCCESS = "File successfully imported. %1$d new contacts added";
+    public static final String MESSAGE_SUCCESS = "File successfully imported from.\n%s new contacts added";
```

Comment:
Perhaps can remove 'from'. Since the sentence will ends at 'File successfully imported from.'